app/views/default.scala.html

```diff
@@ -129,6 +129,18 @@
 
         <div id="site">
         @site
+        <footer class="footer">
+          <div class="container">
+            <p class="text-muted">Build: | @{_root_.notebook.BuildInfo.toMap.toList.sortBy(_._1).map{
+                case (k, Some(v)) => <span>&nbsp;<strong>{k}</strong>-<em>{v}</em>&nbsp;|</span>
+                case (k, None) => <span>&nbsp;<strong>{k}</strong>-<em>Unknown...</em>&nbsp;|</span>
+                case (k, v) if k.trim.matches("^x[A-Z].*$") => <span>&nbsp;<strong>{k.drop(1).head.toString.toLowerCase+k.drop(1).tail}</strong>-<em>{v}</em>&nbsp;|</span>
```

Comment:
because we use keys in sbt like `xHadoopVersion` and so forth. Hence I get rid of this reference to extended conf in the ui.